src/scalper.ts

```diff
@@ -283,7 +295,7 @@ export class Scalper {
         // Check every second if further Delta Hedging required
         if (Math.abs(hedgeDeltaTotal * fairValue) < (this.minSize * fairValue)) {
```

Comment:
how does this work? does sleepRandom update the values in this calculation, or will it always get the same result every time?

Reply:
The deltaFillListener returns a new hedgeDeltaTotal upon receiving a matched fill orderID. By checking each ~1sec we will preserve the twap nature of the delta hedge but allow for fills to be caught by the websocket

Reply:
Makes sense. Didnt see that the hedgeDeltaTotal gets updated somewhere else async